src/app/course-page/course-page.component.spec.ts

```diff
@@ -9,18 +9,23 @@ import { MatInputModule } from '@angular/material/input';
 import { MatDialogModule } from '@angular/material/dialog';
 
 // Directives
-import { BorderHighlightModule } from '../../common/directives/border-highlight/border-highlight.module';
+import { BorderHighlightModule } from '../common/directives/border-highlight/border-highlight.module';
```

Comment:
It will be a good idea to add import resolve paths;
```json
{
  "compilerOptions": {
    "paths": {
      "@app-common": [
        "./src/app/common/"
      ],
      "@app-common/*": [
        "./src/app/common/*"
      ]
    }
  }
}
```
and in other palce in app you will ba ble import anything from common using @app-common
```tsscript
import {somthing} form '@app-common/somthins/somthins'
```